code/modules/mob/living/carbon/human/life.dm

```diff
@@ -32,6 +32,12 @@
 		return
 	//Update our name based on whether our face is obscured/disfigured
 	name = get_visible_name()
+	//makes sure admins are adminning instead of having fun grrr
+	if(ckey == "Silicons" || "CygnusB" || "Putnam" || "R3dtaile" || "DeltaFire" || "R0-adic" || "Xantholne" || "Trojancoyote" || "Dzahlus" || "NicoDeimos" || "Bhijn" ) // Don't know if nico is still an admin, don't have an admin list anywhere so feel free to add if there's people missing
```

Comment:
this will always be true because non-empty strings are truthy, here's the proper way to do it
```suggestion
	if(ckey in list("Silicons", "CygnusB", "Putnam", "R3dtaile", "DeltaFire", "R0-adic", "Xantholne", "Trojancoyote", "Dzahlus", "NicoDeimos", "Bhijn")) // Don't know if nico is still an admin, don't have an admin list anywhere so feel free to add if there's people missing
```

Reply:
thank you mister admin really cool i will be working on it asap